integrations/access/opsgenie/client.go

```diff
@@ -228,6 +228,13 @@ func (og Client) getResponders(reqData RequestData) []Responder {
 	return responders
 }
 
+func (og Client) getPriority(reqData RequestData) string {
+	if priority, ok := reqData.SystemAnnotations[types.TeleportNamespace+types.ReqAnnotationPriority]; ok {
+		return strings.Join(priority, "")
```

Comment:
What's the impact of having P2P1? Will opsgnie fail?
Should we block the priority to have a single value?

Reply:
Yes, the request to Opsgenie fails with `422`.  

```json
{
	"message": "Request body is not processable. Please check the errors.",
	"errors": {
		"priority": "should be one of [ P1, P2, P3, P4, P5 ]"
	},
	"took": 0.0,
	"requestId": "f9a82f83-42b3-4c34-9a7c-2be5c14ed1d9"
}
```

Opsgenie API is already doing that validation so I think we dont need to take care of that. What do you think?

Reply:
It might be simpler to check if `len(...) == 1` and return an error if it's not. This would make it clearer to users that they are sending two different priorities, rather than an incorrect value.